ddtrace/span.py

```diff
@@ -490,7 +491,11 @@ def _remove_exc_info(self):
         self._remove_tag(ERROR_TYPE)
         self._remove_tag(ERROR_STACK)
 
+    @deprecated(version="1.0.0")
```

Comment:
There used to be a removal message here no?

Reply:
Yup, I was just rebasing commits and it got lost in the shuffle

Reply:
Everything should be in order now